siac/hostcmd.go

```diff
@@ -62,6 +62,14 @@ func hostconfigcmd(param, value string) {
 		p.Mul(p, big.NewRat(1e24/1e9, 4320))
 		value = new(big.Int).Div(p.Num(), p.Denom()).String()
 	}
+	// parse sizes of form 10GB, 10TB, 1TiB etc
+	if param == "totalstorage" || param == "minfilesize" || param == "maxfilesize" {
+		var err error // must be pre-declared because value is
```

Comment:
This is actually not true, and an unnecessary line, but at this point I don't know if I should make another commit just for that